### PR TITLE
Adding support for Poet

### PR DESCRIPTION
**Pull request by leek**
Simply adding support for @awendt's [Poet](https://github.com/awendt/poet) tool.

### CODE REVIEW

File: mackup/applications/poet.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = Poet
+
+[configuration_files]
+.ssh/config.d
```
Comment (vitorgalvao) [inline mackup/applications/poet.cfg]: Wrong file.

Comment (leek) [inline mackup/applications/poet.cfg]: **@vitorgalvao:** Huh?

Comment (vitorgalvao) [inline mackup/applications/poet.cfg]: @leek Never mind. My mistake.

**Comment by lra** [comment]
huh, sorry but this will lead to a lot of confusion.
`.ssh` is supposed to be owned by `ssh`, and this looks like a script hacking around its config files.
The problem is that if someone syncs `.ssh` it will mess things up.
This folder `.ssh/config.d` and everything in it should really be owned by `ssh`.

**Comment by lra** [comment]
not sure how to solve this.
Move `.ssh/config.d` in `ssh`?

**Comment by leek** [comment]
@lra I don't see any harm in having `.ssh/config.d` moved into `.ssh`.

**Comment by leek** [comment]
I'm fine with this being in a custom `.cfg` for now. Closing this PR.

**Pull request closed without merging**